jwst/datamodels/model_base.py

```diff
@@ -80,10 +77,6 @@ def __init__(self, init=None, schema=None, extensions=None,
             an excption. If false, they will generate a warning.
         """
         # Set the extensions
-        if extensions is None:
-            extensions = jwst_extensions[:]
-        else:
-            extensions.extend(jwst_extensions)
         self._extensions = extensions
```

Comment:
Based on a quick review of the places where this class is used, it seems like it's no longer necessary to carry along this `extensions` argument at all. That would mean it could be removed from all of the calls to `AsdfFile` below as well. Are there any cases where a different set of extensions other than those provided by the entry points will be needed?